PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/common/AzureComboBox.java

```diff
@@ -121,6 +121,10 @@ public void setValue(final ItemReference<T> val) {
         this.refreshValue();
     }
 
+    protected ItemReference<T> getItemReference() {
+        return this.value instanceof ItemReference ? (ItemReference<T>) this.value : null;
+    }
+
```

Comment:
remove this method. and use `this.value` directly in subclasses.

Reply:
Yes, will set the `ItemReference` in `AppServiceComboBox`, and remove the `getItemReference` in `AzureComboBox`